region/maridia/inner-yellow.json

```diff
@@ -1397,24 +1397,50 @@
       "enemies": [
         {
           "id": "e1",
-          "groupName": "Butterfly Room Zoas",
+          "groupName": "Butterfly Room Right Zoas",
           "enemyName": "Zoa",
-          "quantity": 3,
-          "homeNodes": [ 3 ],
+          "quantity": 2,
+          "homeNodes": [ 2 ],
           "farmCycles": [
             {
               "name": "Farm 2 Zoas out of 3",
               "cycleFrames": 225,
-              "requires": [],
-              "devNote": "Realistically could require a suit or hjb or something, but that's already needed to get in or out of the node"
+              "requires": []
             },
             {
               "name": "Grapple",
               "cycleFrames": 175,
-              "requires": ["Grapple"],
-              "devNote": "Technically doable without reaching node 3, but oh well"
+              "requires": ["Grapple"]
             }
           ]
+        },
+        {
+          "id": "e2",
+          "groupName": "Butterfly Room Left Zoa",
+          "enemyName": "Zoa",
+          "quantity": 1,
+          "homeNodes": [ 1 ],
+          "farmCycles": [
+            {
+              "name": "Farm 1 Zoa out of 3",
+              "cycleFrames": 225,
+              "requires": []
+            },
+            {
+              "name": "Grapple",
+              "cycleFrames": 175,
+              "requires": ["Grapple"]
+            }
+          ]
+        }
+      ],
+      "reusableRoomwideNotable": [
+        {
+          "name": "Butterfly Room Careful Jump",
+          "note": [
+            "Kill all the Zoas and don't pick up their drops. If the door is unlocked, stand in the doorframe and jump across the room.",
+            "Otherwise stand 1.5 tiles away from the sandfall and jump across. Samus will dip into the sand, but still make it to the other side."
```

Comment:
`jump from 1.5 tiles away`  instead of `stand`
It looks like a running start doesn't affect the jump at all.  Straight jumping also works 🗒️